packages/components/anchor/src/Anchor.tsx

```diff
@@ -36,6 +36,14 @@ export default defineComponent({
     const { activeLink } = useLinks(props, config)
     const { anchorRef, inkBallElRef, inkBallClasses, inkBallTop } = useInkBall(activeLink, mergedPrefixCls)
 
+    const affixRef = ref<typeof IxAffix | null>(null)
```

Comment:
```suggestion
    const affixRef = ref<AffixInstance | null>(null)
```

记得给 AffixInstance 加一个 update 的 bindings